docs/integration/OpenAPI.md

```diff
@@ -74,4 +74,4 @@ components {
 
 You can enable logging invocations in the runtime by setting logger
```

Comment:
Do we need this term _invocations_ here? 

Why not just _you can enable enricher level runtime logging by ...._  ?

Reply:
done